hoomd/box.py

```diff
@@ -3,17 +3,15 @@
 import hoomd._hoomd as _hoomd
 
 
-def _make_vec_three(vec, vec_factory, scalar_type):
-    """A helper function for converting Python sequences to HOOMD T3 classes (e.g. Scalar3, Int3).
+def _make_vec3(vec, vec_factory, scalar_type):
+    """Converts Python types to HOOMD T3 classes (e.g. Scalar3, Int3).
 
     Args:
-        vec (Sequence[X] or X): A sequence of type X where X is the type
-            returned by scalar_type.
+        vec (Sequence[T] or T): A sequence or scalar of type ``scalar_type``.
```

Comment:
@b-butler I couldn't find any cases where the 'scalar' form of this function is used. Could you check me on this claim? If my understanding is accurate, should it be simplified to only handle arrays of length 3? It would simplify all the type checking.